sysdig/resource_sysdig_secure_cloud_auth_account_test.go

```diff
@@ -97,7 +97,7 @@ resource "sysdig_secure_cloud_auth_account" "gcp_config_posture" {
 	}
 	secure_identity_entitlement {
 	  enabled    = true
-	  components = ["COMPONENT_SERVICE_PRINCIPAL/secure-posture"]
+	  components = ["COMPONENT_WEBHOOK_DATASOURCE/secure-runtime"]
```

Comment:
this is already fixed up on master

Reply:
Yes, this PR got pushed before the fix. Rebased it.